sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/SpannerWriteIT.java

```diff
@@ -116,9 +117,12 @@ public void setUp() throws Exception {
   }
 
   private String generateDatabaseName() {
-    String random = new RandomStringGenerator.Builder().build()
-        .generate(MAX_DB_NAME_LENGTH - 1 - options.getDatabaseIdPrefix().length())
-        .toLowerCase();
+    String random =
+        new RandomStringGenerator.Builder()
+            .filteredBy(CharacterPredicates.LETTERS)
```

Comment:
I think it would be something like `DIGITS.or(LETTERS)` but I didn't see a trivial way to combine them. So this is overconstrained, but it will be all letters so it should work, and should be random enough.